### PR TITLE
companies index view

### PR DESCRIPTION
**Pull request by hieult-0009**
Config model company
View all companies
Add some i18n
Refactor folder view

### CODE REVIEW

**Comment by annd-0005** [comment]
test chạy thử database CRUD data chưa chú?

**Comment by sinhld-0016** [comment]
code đẹp quá

File: src/main/webapp/WEB-INF/configs/spring-client-service.xml [hunk 1]
```diff
@@ -20,6 +20,7 @@
 		</property>
 	</bean>
 	
+<<<<<<< HEAD
```
Comment (sinhld-0016) [inline src/main/webapp/WEB-INF/configs/spring-client-service.xml]: conflict kìa

File: src/main/webapp/WEB-INF/configs/spring-client-service.xml [hunk 2]
```diff
@@ -28,4 +29,13 @@
...
 
+=======
+	<bean id="companiesService" parent="baseTransactionProxy">
+		<property name="target">
+			<bean class="vn.sun.services.client.impl.CompaniesServicesImpl">
+				<property name="companiesDAO" ref="companiesDAO" />
+			</bean>
+		</property>
+	</bean>
+>>>>>>> companies index view
```
Comment (sinhld-0016) [inline src/main/webapp/WEB-INF/configs/spring-client-service.xml]: nữa nè

**Comment by hieult-0009** [comment]
> test chạy thử database CRUD data chưa chú?
mới load data cùi bắp ra thôi
image

File: src/main/webapp/WEB-INF/configs/spring-client-service.xml (hunk 1 above)
Comment (hieult-0009) [reply src/main/webapp/WEB-INF/configs/spring-client-service.xml]: fixed :v

**Comment by hieult-0009** [comment]
ready

**Pull request merged**